scripts/create-package/plopfile.ts

```diff
@@ -233,12 +233,13 @@ function replaceVersionsFromReference(
 
 function updatePackageJson(packageJsonContents: string, answers: Answers) {
   const { target, hasTests, publish } = answers;
+  console.log(answers);
 
   // Copy dep versions in package.json from actual current version specs.
   // This is preferable over hardcoding dependency versions to keep things in sync.
   // The reference package(s) may need to be updated over time as dependency lists change.
   const newPackageJson: PackageJson = JSON.parse(packageJsonContents);
-  const referencePackages = target === 'node' ? ['codemods'] : ['react-button', 'react-image'];
+  const referencePackages = target === 'node' ? ['codemods'] : ['react-menu', 'react-cards'];
```

Comment:
I am not entirely sure if react-cards is the best here since I have been told it was never intended to be a converged component... @ecraig12345  might have context on this

Reply:
Ok. My intention was to use only `react-menu` as base, but apparently `@fluentui/react-compose` is a required dependency for the `create-component` to identify the generated package. And since `react-menu` doesn't have `@fluentui/react-compose`  I just added `react-cards`.

Would you have a better solution? @ling1726  @ecraig12345

Reply:
oh yeah on that note, no converged package should have a dependency on `react-compose` anymore.

`@fluentui/react-utils` is the package that replaces it but will soon undergo a naming change too 

https://github.com/microsoft/fluentui/pull/16970#issuecomment-778298573